### PR TITLE
Update script-topics.rpy

### PR DESCRIPTION
**Pull request by ShnitzWasTaken**
2 quick topics I came up with

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4782,3 +4782,42 @@ label monika_asks_family:
...
+    m 1a "It might not work all the time but it does help me wake up a little at least."
+    m 4d "Surprising how the absense of a liqid in our body can drag us down so much, huh?"
+    m 2l "...Well, I suppose 70% of what our body is comprised of can sure make a big impact on us!"
+    m 1c "But anyways, make sure to always stay hydrated, ok [player]?"
+    m 4c "In fact..."
+    m 1a "You should probably grab a glass of water right now, actually..."
+    m    "I don't mind waiting a minute or two."
+    m 4k "It's not like I'd run away on you!"
+    m    "Ehehe~"
+return
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: indent th `return`s

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4782,3 +4782,42 @@ label monika_asks_family:
                 m 1f "You can tell me about your family when you're ready, [player]." 
                 m 1j "I love you very much!"                              
     return
+
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: You're missing the `init 5 python` blocks that actually add your topic to the game. Refer to [here](https://github.com/Backdash/MonikaModDev#adding-content)

**Comment by ShnitzWasTaken** [comment]
Alright, changed it so it should work properly now

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4782,3 +4782,47 @@ label monika_asks_family:
...
+    m 1a "It might not work all the time but it does help me wake up a little at least."
+    m 4d "Surprising how the absense of a liqid in our body can drag us down so much, huh?"
+    m 2l "...Well, I suppose 70% of what our body is comprised of can sure make a big impact on us!"
+    m 1c "But anyways, make sure to always stay hydrated, ok [player]?"
+    m 4c "In fact..."
+    m 1a "You should probably grab a glass of water right now, actually..."
+    m 1a "I don't mind waiting a minute or two."
+    m 4k "It's not like I'd run away on you!"
+    m 4k "Ehehe~"
+return
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Indent your `return`s.

**Pull request merged**